recipe/build_kaldi.sh

```diff
@@ -54,14 +54,30 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
     fi
 fi
 
-if [[ ${cuda_compiler_version} != "None" ]]; then
+if [[ "${cuda_compiler_version}" =~ 12.* ]]; then
     export USE_CUDA=1
-    export NCCL_ROOT_DIR=$PREFIX
-    export NCCL_INCLUDE_DIR=$PREFIX/include
+    export NCCL_ROOT_DIR="${PREFIX}"
+    export NCCL_INCLUDE_DIR="${PREFIX}/include"
     export USE_SYSTEM_NCCL=1
     export USE_STATIC_NCCL=0
     export USE_STATIC_CUDNN=0
-    export CUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME
+    export CUDA_TOOLKIT_ROOT_DIR="${PREFIX}"
+    export MAGMA_HOME="${PREFIX}"
+    if [[ "${target_platform}" == "linux-64" ]]; then
+        export NVTX_ROOT_DIR="${PREFIX}/targets/x86_64-linux"
```

Comment:
As both [the NVTX headers and the libraries are in this directory on CUDA 12 (Linux `x86_64`)]( https://github.com/conda-forge/cuda-nvtx-feedstock/blob/ba84e9b7c76c4bb6990002b7393731c79ab0e501/recipe/meta.yaml#L75-L77 ), had tried to point it here, but that didn't seem to work as expected

Maybe Kaldi has a better way to specify this location?